keyboards/ducky/one2mini/matrix.c

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
What has changed here, compared to the standard implementation in QMK?
Can this just be deleted?

Reply:
Or if its a hard requirement, can it be refactored to use custom matrix lite.

Reply:
Personally, I'm reluctant to accept "here and there" as a good enough reason to go towards custom matrix (Given it introduces an extra layer of maintenance burden).